packages/contractkit/src/wallets/signers/rpc-signer.ts

```diff
@@ -0,0 +1,75 @@
+// @ts-ignore-next-line
+import { account as Account } from 'eth-lib'
+import * as ethUtil from 'ethereumjs-util'
+import { InterceptedMethods } from '../../providers/celo-provider'
+import { RpcCaller } from '../../utils/rpc-caller'
+import { RLPEncodedTx } from '../../utils/signing-utils'
+import { Signer } from './signer'
+
+const decodeSig = (sig: any) => {
```

Comment:
Thanks for modularizing this. Can you move this to `signing-utils.ts` and update `local-signer` to use this?